asclepius/asclepius/enrich.py

```diff
@@ -1,8 +1,40 @@
-def enrich(ddis):
-    def _enrich(ddi):
-        result = {'harmful': bool(hash(ddi['uri']) % 2),
-                  'spacing': abs(hash(ddi['uri']))%14 + 1}
-        result.update(ddi)
-        return result
+from hashlib import sha1
+import csv
+import sys
+import dinto
+import clize
+from clize.parameters import one_of
+
+def _sha_int(string):
+    return int(sha1(string.encode()).hexdigest()[-8:], 16)
```

Comment:
That would probably work too. I was suggesting `int(string, base)` where `base` is like 256 (something big enough to account for all valid `uri` chars). Basically pretend the `uri` is some base 256 number.

Whatever you want to go with.